.github/workflows/test.yml

```diff
@@ -27,12 +27,6 @@ jobs:
           terraform_version: '1.5.*'
           terraform_wrapper: false
 
-      - name: Run linters
-        uses: golangci/golangci-lint-action@639cd343e1d3b897ff35927a75193d57cfcba299 # v3.6.0
-        with:
-          version: latest
-          args: --timeout 3m
-
```

Comment:
this is removed because code analysis got extracted into a separate job thus this not not needed here anymore

Reply:
Ack.